src/bindings/python/src/compatibility/ngraph/opset9/ops.py

```diff
@@ -48,6 +49,48 @@ def eye(
     return _get_node_factory_opset9().create("Eye", inputs, {"output_type": output_type})
 
 
+def roi_align(
+    data: NodeInput,
+    rois: NodeInput,
+    batch_indices: NodeInput,
+    pooled_h: int,
+    pooled_w: int,
+    sampling_ratio: int,
+    spatial_scale: float,
+    mode: str,
+    aligned_mode: Optional[str] = "asymmetric",
+    name: Optional[str] = None,
+) -> Node:
+    """Return a node which performs ROIAlign operation.
+
+    :param data: Input data.
+    :param rois: RoIs (Regions of Interest) to pool over.
+    :param batch_indices: Tensor with each element denoting the index of
+                          the corresponding image in the batch.
+    :param pooled_h: Height of the ROI output feature map.
+    :param pooled_w: Width of the ROI output feature map.
+    :param sampling_ratio: Number of bins over height and width to use to calculate
+                           each output feature map element.
+    :param spatial_scale: Multiplicative spatial scale factor to translate ROI coordinates.
+    :param mode: Method to perform pooling to produce output feature map elements.
+    :param aligned_mode: specifies how to transform the coordinate in original tensor to the resized tensor.
+                         Mode 'asymmetric' is default.
```

Comment:
```suggestion
    :param mode: Method to perform pooling to produce output feature map elements.
    :param aligned_mode: Specifies how to transform the coordinate in original tensor to the resized tensor.
                         Mode 'asymmetric' is the default value. Optional.
```

Reply:
You may also specify what values of `mode`/`aligned_mode` are possible.

Reply:
great idea!